easybuild/easyconfigs/s/squashfs-tools/squashfs-tools-4.6.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'squashfs-tools'
+version = '4.6.1'
+
+homepage = ' https://github.com/plougher/squashfs-tools'
+description = """squashfs-fuse"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/plougher/squashfs-tools/archive']
+sources = ['master.tar.gz']
```

we try to avoid using a branch as much as possible for reproducibility, always use a release, a tag, or a commit:
```suggestion
sources = ['%(version)s.tar.gz']
```

you also have to add a checksum.
simplest way to do that is with `eb squashfs-tools-4.6.1-GCCcore-12.3.0 --inject-checksums`